install/kubernetes/helm/meshery/charts/meshery-kuma/templates/ingress.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "meshery-kuma.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
```

Comment:
Same as above `Values.service.port` can be find in `values.yaml`

Reply:
Yes, and for that I do not see why we define additional variable as "$svcPort" and then we use it. You could directly use it because you do not change the current scope.